admin-ui/src/pages/settings-page.tsx

```
@@ -806,16 +988,156 @@ function ExtraPromptsCard({
+                      </Select>
+                      <Button
+                        type="button"
+                        variant="outline"
+                        size="sm"
+                        onClick={() => onRemoveItem(item.id)}
+                      >
+                        Remove
+                      </Button>
+                    </div>
+                    <div className="text-muted-foreground text-xs">
+                      Requests can use either the alias or the original model name.
```

ESLint

[error] Insert `⏎·····················`

(prettier/prettier)